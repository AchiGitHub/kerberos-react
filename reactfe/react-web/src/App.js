import React, { Component } from 'react';
import Login from './modules/login/login';
import {
  BrowserRouter as Router,
  Route
} from 'react-router-dom'
import Home from './modules/login/Home';

class App extends Component {
  render() {
    return (
      <Router>
        <Route exact path="/" component={Login}/>
        <Route path="/home" component={Home}/>
      </Router>
    );
  }
}

export default App;
