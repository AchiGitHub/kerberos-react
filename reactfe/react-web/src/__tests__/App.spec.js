import React from 'react';
import { mount, shallow, configure } from 'enzyme';
import App from '../App';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('App >> connected componenet', () => {
    let container

    it('should render the App component', () => {
        container = shallow(<App />)
        expect(container).toMatchSnapshot()
    });
})