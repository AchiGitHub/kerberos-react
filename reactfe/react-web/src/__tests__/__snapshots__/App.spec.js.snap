// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App >> connected componenet should render the App component 1`] = `ShallowWrapper {}`;
